src/SearchCallbacks/Equals.php

```diff
@@ -42,10 +42,27 @@ public function execute(Builder $builder, string $column, CategorizedValues $val
         }
 
         if ($values->and) {
-            if ($this->isDate($this->searchParser->type) || $this->isDateTime($this->searchParser->type)) {
+            if ($this->isDate($this->searchParser->type)) {
                 foreach ($values->and as $andValue) {
                     $builder->orWhereDate($column, $andValue);
                 }
+            } elseif ($this->isDateTime($this->searchParser->type)) {
+                foreach ($values->and as $andValue) {
+                    $dateTimeValue = new \DateTime($andValue);
+                    $formattedDateTime = $dateTimeValue->format('Y-m-d H:i:s');
+
+                    if ($dateTimeValue->format('s') === '00') {
+                        $builder->orWhere(function ($query) use ($column, $formattedDateTime) {
+                            $query->whereDate($column, '=', date('Y-m-d', strtotime($formattedDateTime)))
+                                  ->whereTime($column, '>=', date('H:i', strtotime($formattedDateTime)))
+                                  ->whereTime($column, '<', date('H:i', strtotime($formattedDateTime . ' +1 minute')));
```

Comment:
and what if we strictly want to search by second = 0? :)

Reply:
you're right, someone could search for '000' or '0000' as well :)) so i'll change this to
if (preg_match('/^0+$/', $dateTimeValue->format('s'))) {...}

Reply:
Not needed, because it doesn't change things.
Seconds will always be 00 in this case, number of 0 was not the point... :)
The point is that FE only selects up to minutes (and adds 00 as seconds) - so this scenario works

Reply:
If the number of zeros wasn't the point, why mention it ;) Since it was mentioned, I went one step further and satisfied those who would go crazy entering endless zeros ;)

Reply:
I did not mention number of zeroes, but the fact that you can not find a record that strictly has 0 second. :)
for example: 2024-07-11 11:28:00

;)

Reply:
I think we both understood each other in the first run and that's all that matters ;)